package.json

```diff
@@ -137,5 +140,28 @@
     "tslint": "~6.1.0",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "~4.0.3"
+  },
+  "gmailLibrary": {
```

Comment:
Could you remove this part it looks unnecessary since you already have declared the library as `"gmail-tester-extended": "^1.1.2",` and the original author is just Lev Gelfenbuim

Reply:
Hello Daniel, could you please clarify which line numbers you consider unnecessary? thank you!